WordPress-iOS-Shared/Core/WPTableViewSectionHeaderFooterView.m

```diff
@@ -0,0 +1,227 @@
+#import "WPTableViewSectionHeaderFooterView.h"
+#import "WPTableViewCell.h"
+#import "WPStyleGuide.h"
+#import "NSString+Util.h"
+
+
+
+@interface WPTableViewSectionHeaderFooterView ()
+@property (nonatomic, strong) UILabel *titleLabel;
+@end
+
+
+@implementation WPTableViewSectionHeaderFooterView
+
+- (instancetype)initWithStyle:(WPTableViewSectionStyle)style
+{
+    NSParameterAssert(style == WPTableViewSectionStyleHeader || style == WPTableViewSectionStyleFooter);
+    
+    self = [super init];
```

Comment:
Ahhhhhh awesome catch, i absolutely missed that. Updating accordingly, thanks sergio!